src/components/UserDashboard/index.js

```diff
@@ -55,21 +56,25 @@ export default compose(
           padding: 40px;
         `}
       >
-        <H2 pb={10}>Welcome, {loggedInUser.firstName}!</H2>
+        <H1 mb={'35px' /* On removing Beta Message change to 44px */}>
+          Welcome, {loggedInUser.firstName}!
+        </H1>
         <Column>
           <PromptMessageContainer info my={20}>
             <PromptMessageHeading info mb={10} color={theme.secondary}>
-              The Portal is currently in <strong>BETA Phase</strong>
+              The Portal is currently in <H2 display="inline-block">BETA Phase</H2>
```

Comment:
just a nit-pick, not sure I entirely agree with the semantic of having a header inside a header, isn't the whole thing a header and this part is just bolded? #problemForLater :P